lib/api/route/messages.dart

```diff
@@ -263,3 +263,60 @@ class UploadFileResult {
 
   Map<String, dynamic> toJson() => _$UploadFileResultToJson(this);
 }
+
+/// https://zulip.com/api/add-reaction
+Future<AddReactionResult> addReaction(ApiConnection connection, {
+  required int messageId,
+  required ReactionType reactionType,
+  required String emojiCode,
+  required String emojiName,
+}) {
```

Comment:
Maybe `addReaction` and `removeReaction` could take a `Reaction` object instead of `reactionType`, `emojiCode`, and `emojiName` individually. Would that be better, I wonder?

Reply:
Yeah, I agree that wouldn't be better, because the `userId` wouldn't belong.